src/state/machines/turn.ts

```diff
@@ -1002,15 +1041,6 @@ export const TurnMachine = setup({
         },
         refreshing: {
           on: {
-            "user.click.player.hand.card.ability": {
```

Comment:
After testing this locally I think we might have to bring it back to some extent. I specifically made it like this so that you can change your mind when using an ability on a card with multiple abilities. This allowed you to basically deselect the currently selected ability and select another one all without having to reactivate the card ability. Now when you click on the ability the whole card ability is cancelled